docs/how-it-works/features/security/audit-cloudtrail.md

```diff
@@ -1,32 +1,41 @@
 # Audit | CloudTrail
 
-AWS CloudTrail monitors and records account activity across your AWS infrastructure, giving you control over storage, analysis, and remediation actions.
+## Feature Overview 
 
+AWS CloudTrail monitors and records account activity across your AWS infrastructure, 
+giving you control over storage, analysis, and remediation actions.
 
-![Cloudtrail Diagram](../../../assets/images/diagrams/aws-cloudtrail.svg)
-
-!!! note ":bookmark_tabs: [Leverage Reference Architecture | Security Audit](https://github.com/binbashar/le-tf-infra-aws/tree/master/apps-prd/us-east-1/security-audit)"
-
-**CloudTrail module**. Key elements:  
-
-* Destination bucket
-* KMS Key to files encrypt files
-* Include global services
-* Enable multi-regional trail
-
-
-**S3 Bucket Module**. Key elements:
+!!! info "AWS CloudTrail overview"
+    This service will be configured to enable auditing of all AWS services in all accounts.
+    Once enabled, as show in the below presented figure CloudTrail will deliver all events 
+    from all accounts to the Security account in order to have a centralized way to audit 
+    operations on AWS resources.
+    Audit events will be available from CloudTrail for 90 days but a longer retention 
+    time will be available through a centralized S3 bucket.
 
-* Lifecycle rule
-* Expiration
-
-**KMS Key Module**. Key elements:
-
-* Deletion Window
-* Policy  
-  *Grant permission to the rest of the accounts to use the key*
-
-# Read More
-
-* :cloud: [AWS Cloudtrail Overview](https://aws.amazon.com/cloudtrail/)
-* :notebook: [Security Pillar - AWS Well-Architected Framework](https://docs.aws.amazon.com/wellarchitected/latest/security-pillar/welcome.html)
\ No newline at end of file
+![Cloudtrail Diagram](../../../assets/images/diagrams/aws-cloudtrail.svg)
```

Comment:
I was about to suggest centering this diagram, and actually attempted to do it on my machine using https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions to no avail.
So I guess it will stay that way 😅

Reply:
@angelofenoglio this will be great, I've seen that the official example works perfectly, I'll give it a try and let you know.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/18539704/178320782-8297d462-8343-4799-9b6d-93886a9c8c2e.png">

Reply:
@angelofenoglio it worked :) We could then propagate this improvement for the rest of the doc figures too.

<img width="501" alt="image" src="https://user-images.githubusercontent.com/18539704/178321215-23ac1728-9cca-4806-b40d-62634c3cd2ec.png">